build.gradle

```diff
@@ -1459,7 +1459,7 @@ project(':group-coordinator:group-coordinator-api') {
   }
 
   task createVersionFile() {
-    def receiptFile = file("$buildDir/kafka/$buildVersionFileName")
+    def receiptFile = file("${layout.buildDirectory.get().asFile}/kafka/$buildVersionFileName")
```

Comment:
do we need the path here instead since we want to use a string and not the java.io.File object?

Perhaps, `layout.buildDirectory.get().asFile.path`

(similar comment for other places)